### PR TITLE
homework_1

### PR DESCRIPTION
**Pull request by websafarispb**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,62 +12,89 @@
...
+        <version>3.0.0-M1</version>
+        <configuration>
+          <suiteXmlFiles>${suite.file}</suiteXmlFiles>
+        </configuration>
+      </plugin>
+    </plugins>
   </build>
+  <profiles>
+  <profile>
+    <id>default</id>
```
Comment (DmitryKhodakovsky) [inline pom.xml]: Why is default **maven profile** with all tests? Please read carefully homework description

Also `default` is not good name for profile. It does not clear which test could be running thought it

File: src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package ru.stepev.test.training.at.hw1;
+
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+import ru.stepev.test.training.at.hw1.dataprovider.DivideDataProvider;
+
+public class CalculatorDivideTest extends CalculatorTest {
+
+    public static final double DELTA = 0.01;
```
Comment (DmitryKhodakovsky) [inline src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java]: I suppose that `DELTA` could be extracted to the `CalculatorTest` class as `protected` variable

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Please resolve comments

File: pom.xml (hunk 1 above)
Comment (websafarispb) [reply pom.xml]: Sorry, I've changed it.

File: src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java (hunk 2 above)
Comment (websafarispb) [reply src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java]: I've changed it.

**Review by DmitryKhodakovsky — APPROVED** [review]
HW1 approved

**Pull request merged**